vnext/ReactUWP/ChakraPreparedScriptStore.cpp

```diff
@@ -0,0 +1,58 @@
+#include "pch.h"
+#include "ChakraPreparedScriptStore.h"
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Storage.h>
+#include <Utils/LocalBundleReader.h>
+#include "LocalByteCodeManager.h"
+#include "ChakraJsiRuntime.h"
+#include <winrt/Windows.Storage.Streams.h>
+
+using namespace winrt;
+
+namespace react { namespace uwp {
+
+std::unique_ptr<const facebook::jsi::Buffer> ChakraPreparedScriptStore::tryGetPreparedScript(
```

Comment:
lets rename these to UwpPreparedScriptStore. same with the regular script store.

There isn't anything specific to Chakra here. but we are using UWP specific folders and file access.